index.html

```diff
@@ -1,15 +1,384 @@
+            is already stuffed full of high performance, high capacity disks,
+            which means we need to look at replacing it with something more modern
+            and larger.</p>
+
+            <h4 id="secondary">Secondary</h4>
+
+            <p>The secondary machine is going flat-out and the bottleneck is the
+            transfer of data to and from the disks. The machine could keep the
+            tiles more up-to-date if we removed that bottleneck, which means
+            upgrading the RAID controller and adding a mirror disk for the main
+            database.</p>
+
+            <h3 id="imagery-server">Imagery server</h3>
+
+            <p>We know that out-of-copyright (OOC) maps and aerial imagery are both
+            very useful sources for mapping activities, and we’re not happy that
+            the machine which used to serve this data is currently down. The
+            reason is that the large external disk arrays used to store the
+            imagery are very old and consume too much power for the home they are
+            in while our friends at UCL re-build the building which used to be
+            their home.</p>
+
+            <p>To fix this, we can get a smaller, more modern and more power
+            efficient server to serve that data, and get this service back online
+            where it’s useful to everyone.</p>
+
+            <p>(Side note: is it worth / possible looking at dumping the projected
+            images on S3?)</p>
+
+            <h3 id="services-machine-replica">Services machine replica</h3>
+
+            <p>The “services” machine currently runs a mix of stuff, including the
+            planet download site, backups and the internal configuration
+            server. All of these are also backed-up remotely, for safety. However,
+            it would take some time to recover those files from a remote backup,
+            simply due to the greater bandwidth within the server room. We would
+            like to have a “live” backup of this machine to minimise any downtime
+            when the primary machine, or the site it’s hosted at, aren’t
+            available.</p>
+
+            <h3 id="consumables">Consumables</h3>
+
+            <p>Sadly, nothing lasts forever. And hard disks, SSDs, RAID controller
+            batteries, PSUs, fans, cables and many other computer components don’t
+            even come close! Every year, we replace many of these items which have
+            failed or come to the end of their lives. Many are replaced under
+            warranty, but we still have to spend money on replacing or fixing
+            those which are out of warranty or not covered.</p>
+
+            <h2>History</h2>
+           <p>The Operations Working Group has scaled OpenStreetMap since 20XX. We're here to maintain servers day-to-day from applying security patches to fine tuning the stack. From time to time we apply larger upgrades, here are some of the more notable ones.</p>
```

Comment:
It might be okay to ignore this for donation copy, but the OWG [doesn't do the day-to-day running of the servers](http://www.osmfoundation.org/wiki/Operations_Working_Group) - that's the sys admins group.